<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <link rel="stylesheet" href="styles.css">
    <title>Surya Appana's Portfolio</title>
</head>
<body>
    <div class="div-heading">
        <h1 class="heading">Surya Appana's Webpage</h1>
    </div>
    <div class="div-body">
        <div class="div-sub">
            <h3>About</h3>
            <p class="summary">
                I am an undergraduate student studying Applied Mathematics and Data Science at the
                University of California, Berkeley. I am passionate about data science a focus on machine and deep learning
                theory and applications in healthcare, biomedical, and drug contexts. I also enjoy exploring applied math areas
                such as numerical linear algebra. I aim to pursue a career focused in AI for health-centered research.
            </p>
        </div>
        <div class="div-sub">
            <div class="wrapper">
                <h3>Contact</h3>
                <p>Email: sur123app at gmail dot com</p>
                <a href="https://www.linkedin.com/in/surya-a-44a3621aa/" style="color: blue;">LinkedIn</a>
            </div>
        </div>
    </div>
    <div class="div-body">
        <table class="projects-table">
            <caption><h3>Projects and Research</h3></caption>
            <tr>
                <td class="project-title">
                    <b>Genomic Analysis of Pseudonomas alloputida</b>
                    <p>UC Berkeley, Compbio C131 Introduction to Computational Molecular and Cell Biology </p>
                    <p>May 2025 - August 2025</p>
                    <a href="BioE_131_Final_Project.pdf" style="color: blue;">Final Report:</a><i>Genome Analysis of Pseudomonas alloputida</i>
                </td>
                <td class="project-descr">
                    The goal of this project was to fully characterize a Pseudomonas alloputida isolate by assembling its genome, annotating its functional content, reconstructing its metabolic pathways, and placing it within an evolutionary context.
                    This required several stages: raw read quality control, filtering and trimming, long-read genome assembly, assembly quality assessmenst, genome annotation, metabolic modeling, and comparative phylogenetics. My contributions focused
                    on the early computational pipeline—processing and refining the Nanopore and Illumina sequencing data using FASTQC, Filtlong, Cutadapt, and Trimmomatic, assembling the genome with Flye, and evaluating assembly accuracy and completeness
                    with QUAST and CheckM. These steps produced a high-quality, single-contig assembly that enabled the rest of the team to perform downstream analyses. As a whole, the project revealed that the complete and uncontaminated P.
                    putida–group genome isolate has extensive metabolic versatility and a broad suite of xenobiotic degradation pathways, consistent with an adaptable environmental bacterium found in soil and wastewater systems.
                </td>
                <td style="text-align: center; vertical-align:top;">
                    <img src="images/BioE 131 Final Project.png" style="width: 200px; height: 200px;">
                </td>
            </tr>
            <tr>
                <td class="project-title">
                    <b>Coupling AI Co-scientists with high-performance computing (HPC) for the discovery of metal-organic frameworks (MOFs) for carbon capture</b>
                    <p>Big Data REU: University of Chicago </p>
                    <p>May 2025 - August 2025</p>
                    <a href="Big_Data_REU_2025___SC25_Poster.pdf" style="color: blue;">Poster</a>
                </td>
                <td class="project-descr">
                    I worked with professor Kyle Chard at the University of Chicago on coupling large language model (LLM) agents with MOF discovery tools run on HPC resources. Agents include an LLM for generating SMILES of organic linker molecules,
                    assembler for combining linkers and inorganic metal nodes, molecular dynamics simulation (LAMMPS) for estimating structural stability, and deep-learning model for predicting MOF properties (MOF Transformer). We demonstrated success in the ability of the LLM agents
                    to leverage HPC resources - including multiple compute nodes from the Purdue Anvil supercomputer - to accerate MOF discovery. Findings were presented through a poster at ACM Supercomputing 2025.
                </td>
                <td style="text-align: center; vertical-align:top;">
                    <img src="images/ai_coscientist.png" style="width: 200px; height: 170px;">
                </td>
            </tr>
            <tr>
                <td class="project-title">
                    <b>Topic Modeling of Mental Health Forum Posts for Thematic Analysis</b>
                    <p>National University of Singapore (NUS), Institute of Mathematical Sciences (IMS), Ministry of Health Office of Healthcare Transformation (MOHT)</p>
                    <p>May 2024 - July 2024</p>
                    <a href="PUBLICATION - Digital Mental Health Discourse Among Singaporean Youth.pdf" style="color: blue;">Publication:</a><i> Digital Mental Health Discourse Among Singaporean Youth: A Topic Modeling Analysis and Comparison to International Platforms</i>
                </td>
                <td class="project-descr">
                    I worked with Singapore's Ministry of Health, Office of Healthcare Transformation (MOHT)
                    to research posts in MOHT's mental health forum, specifically to discover themes and topics
                    discussed by users. Topic modeling algorithms researched include Latent Dirichlet Allocation (LDA),
                    Nonnegative Matrix Factorization (NMF), Top2Vec, and BERTopic. A novel clustering method was
                    proposed to account for the stochastic nature of BERTopic, the best performing algorithm on our dataset.
                    A comparative analysis between mental health posts on MOHT's forum, Reddit, and HardwareZone (Singaporean forum)
                    was performed to discover insights into similarities and differences based on type of forum and demographics. The
                    project was presented at the 2025 Joint Mathematics Meetings.
                </td>
                <td style="text-align: center; vertical-align:top;">
                    <img src="images/rips_moht_pic.png" style="width: 200px; height: 170px;">
                </td>
            </tr>
            <tr>
                <td class="project-title">
                    <b>Wildfire Burned Area Segmentation with Satellite Imagery (Research)</b>
                    <p>UC Davis, Professor Ali Moghimi</p>
                    <p>June 2023 - August 2023<p>
                </td>
                <td class="project-descr">I worked on implementing specialized convolutional U-Nets for
                    segmenting burned areas in multispectral Sentinel-2 Level 2A satellite images of California 2020 wildfires.
                    The end goal is to combine methods for active fire detection, burned area segmentation, and
                    smoke detection to devise a single deep learning model for simulateous prediction of
                    in all three aspects. Current progress includes a well-performing baseline U-Net for burned area
                    segmentation using red, green, and blue band images after data augmentation.
                </td>
                <td style="text-align: center; vertical-align:top;">
                    <img src="images/burned_area_segmentation_img.png" style="width: 200px; height: 200px;">
                </td>
            </tr>
            <tr>
                <td class="project-title">
                    <b>Columbia River Chinook Salmon Growth Research with Random Forest Models</b>
                    <p>UC Davis, Baskett Lab</p>
                    <p>January - July 2021<p>
                    <a href="https://github.com/Mathgo12/Chinook-Growth-Project" style="color: blue;">Github Repository</a>
                </td>
                <td class="project-descr">I worked with a PhD student working within the Baskett Lab on
                    machine learning and statistical algorithms for predicting growth of
                    salmon across populations and time in the Columbia River. I used both generalized additive models
                    (GAMs) and Random Forest models to make predictions using Python tools. Random Forest was also
                    used to assess variable importance, where the variables included climate and oceanographic predictors
                    that influence the growth of salmon.
                </td>
                <td style="text-align: center; vertical-align:top;">
                    <img src="images/chinook_project_img.png" style="width: 200px; height: 170px;">
                </td>
            </tr>

        </table>
    </div>
    <div class="div-body">
        <div class="div-sub">
            <h3>Education</h3>
            <p>University of California, Berkeley: 2022 - 2026: <a style="color: blue;"
                    href="Transcript - Pre Fall 2025.pdf" download="transcript">Transcript</a></p>
            <ul>
                <li>Math 54: Linear Algebra and Differential Equations</li>
                <li>Math 55: Discrete Mathematics</li>
                <li>Math 110: Abstract Linear Algebra</li>
                <li>Math 156: Numerical Analysis for Data Science and Statistics</li>
                <li>Math 104: Real Analysis</li>
                <li>Math 113: Abstract Algebra</li>
                <li>Math 126: Partial Differential Equations</li>
                <li>Math 185: Complex Analysis</li>
                <li>Math 128a: Numerical Analysis</li>
                <li>Math 118: Fourier Analysis, Wavelets, and Signal Processing</li>
                <li>Math 228b: Numerical Solutions of Differential Equations</li>
                <li>CS 61a: The Structure and Interpretation of Computer Programs (Python)</li>
                <li>CS 61b: Data Structures (Java)</li>
                <li>CS 170: Efficient Algorithms and Intractable Problems</li>
                <li>CS 189: Machine Learning</li>
                <li>Stat 210a: Theoretical Statistics</li>
                <li>EECS 127: Optimization Models in Engineering</li>
                <li>EECS 126: Probability and Random Processes</li>
                <li>Data C8: Foundations of Data Science</li>
                <li>Data 100: Principles and Techniques of Data Science</li>
                <li>Compbio C131: Introduction to Computational Biology</li>
                <li>Cubstart: Full Stack Website Development (MongoDB, Express.js, Node.js, HTML/CSS/JS)</li>
            </ul>
        </div>
        <div class="div-sub">
            <div class="wrapper">
                <h3>Skills</h3>
                <ul>
                    <li>Machine Learning in Python with Pytorch, Tensorflow, Sci-kit Learn, Numpy, Pandas.</li>
                    <li>Programming Languages: Python, R, Java, Javascript, SQL</li>
                    <li>MATLAB</li>
                    <li>Website Development with HTML, CSS, Javascript and Node.js, Express.js</li>
                    <li>Working with Git and Github</li>
                    <li>LaTeX</li>
                    <li>Tableau for data visualization</li>
                    <li>Communication and research skills</li>
                </ul>
            </div>
        </div>
    </div>
</body>
</html>